src/__swaps__/screens/Swap/resources/_selectors/assets.ts

```diff
@@ -59,8 +51,8 @@ export function selectUserAssetAddressMapByChainId(assets: ParsedAssetsDictByCha
 // selector generators
 export function selectUserAssetWithUniqueId(uniqueId: UniqueId) {
   return (assets: ParsedAssetsDictByChain) => {
-    const { chain } = deriveAddressAndChainWithUniqueId(uniqueId);
-    return assets?.[chain]?.[uniqueId];
+    const { chainId } = getAddressAndChainIdFromUniqueId(uniqueId);
```

Comment:
in `deriveAddressAndChainWithUniqueId` we assumed the `uniqueId` has `chainId` as the suffix; in  `deriveAddressAndChainWithUniqueId`, we assume the `uniqueId` has `network` as the suffix

Reply:
Are there any instances where we want to keep `network` as the suffix now? If not I'll just derive chainId from both paths in this function.